ci/build_container/update_build_container.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+read -r -p "Do you have master checked out with most recent changes? [y/N] " response
+if [[ $response =~ ^([yY][eE][sS]|[yY])$ ]]
+then
+  TAG="$(git rev-parse origin/master)"
```

Comment:
this should be master, not origin/master, as your local my not be up to date and that is what you are building.